### DISCUSSION TITLE
Admonitions

### DISCUSSION THREAD

**Discussion by erwinlauener** [thegeeklab/hugo-geekdoc#717] [Q&A] [answered]
Is there a possibility to use collapsable admonitions?

**Comment by xoxys** [comment]
I don't know what collapsible admonitions are. While using asciidoc with Hugo, I'm only aware of the default[ asciidoc admonitions ](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/) which are not collabsible.

**Comment by erwinlauener** [comment]
Thanks for your answer. Here are examples:Have a look at the type ‚note‘Shortcodeshugo-theme-even.netlify.appOr within the Goldmark renderer:admonitions package - github.com/stefanfritsch/goldmark-admonitions - Go Packagespkg.go.devOr Admonitions extreme😉Theme Documentation - Extended Shortcodeshugodoit.pages.devHope this helpsAm 11.10.2023 um 20:32 schrieb Robert Kaussow ***@***.***>:﻿
I don't know what collapsible admonitions are. While using asciidoc with Hugo, I'm only aware of the default asciidoc admonitions  which are not collabsible.

**Reply by xoxys** [reply, marked as answer]
No, the theme's short code for [hints](https://geekdocs.de/shortcodes/hints/) is not collapsible, but you can add custom short codes easily in your page project.

**Comment by erwinlauener** [comment]
Thx 🙏 Am 11.10.2023 um 21:15 schrieb Robert Kaussow ***@***.***>:﻿
No, the theme's short code for hints is not collapsible, but you can add custom short codes easily in your page project.

**Comment by erwinlauener** [comment]
Is the [Collapsible Block](https://docs.asciidoctor.org/asciidoc/latest/blocks/collapsible/#ex-collapsible-paragraph) (AsciiDoc) next on your Roadmap? This would be a solution for me...

**Reply by xoxys** [reply]
Collapsible blocks in asciidoc are working out of the box, as this is handled by the asciidoc renderer.

![image](https://github.com/thegeeklab/hugo-geekdoc/assets/3391958/cded4844-0918-4539-b531-7e72335336db)

**Comment by erwinlauener** [comment]
Ui, I‘ll give it a try🙂Am 12.10.2023 um 10:13 schrieb Robert Kaussow ***@***.***>:﻿
Collapsible blocks in asciidoc are working out of the box, as this is handled by the asciidoc renderer.
